packages/components/src/components/comparison-table/table.stories.tsx

```diff
@@ -0,0 +1,141 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { hiveThemeDecorator } from '../../../../../.storybook/hive-theme-decorator';
+import { ComparisonTableProps, ComparisonTable as Table } from './index';
+
+export default {
+  title: 'Components/Table',
+  component: Table,
+  decorators: [hiveThemeDecorator],
+  parameters: {
+    padding: true,
+  },
+  argTypes: {
+    scheme: {
+      control: 'select',
+      options: ['green', 'neutral'],
+    },
+  },
+} satisfies Meta<ComparisonTableProps>;
+
+export const PerformanceTable: StoryObj<ComparisonTableProps> = {
+  render: props => (
+    <Table
+      scheme="neutral"
+      // highlight can be set with CSS for the cases where it's awkward to add it as a prop
+      // `initial` isn't a perfect name for `"yes"`, but this is an edge case for MDX anyway
+      className="w-min whitespace-pre [&_tbody_tr:last-child]:[--highlight:initial]"
```

Comment:
I'd rather have a `tbody tr:last-child` selector and `--highlight:initial` like this in the code than `any` typed props and children strings tested with regex.